content/source/docs/enterprise/organizations/membership.html.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: "enterprise"
+page_title: "Membership - Organizations - Terraform Enterprise"
+sidebar_current: "docs-enterprise-organizations-membership"
+description: |-
+  Terraform Enterprise allows collaboration in organizations. However, there are several membership levels within organizations.
+---
+
+# Membership
+
+Terraform Enterprise allows collaboration in organizations. However, there are several membershio levels within organizations.
+
+### Admin
+
+Admin is the top-most access level within Terraform Enterprise and is reserved for Terraform install admins. Admins are able to manage users, organizations, and jobs. For example, the folks on our support team have admin privileges that allow them to clear out stuck plans within Terraform environments or to impersonate accounts to view organization data in order to resolve help requests. Admins do not have access to sensitive variables.
+
+### Owner
+
+Owners are the top-most access level within an organization. Owners have complete administrative ability. An owner is able to manage teams and memberships. Owners can also enforce two factor authentication across the organization.
+
+The owners team is a special team made up of all the folks that have `owner` permissons. The team has implied access for all of your resources, but also has the ability to manage your organization.
+
+Owners can also restrict access to resources (Terraform Environments, Packer Builds, and Artifacts). To manage access go to the Access settings for the specific resource and enter the team or collaborators name. At this point you can control the access level for that team: either `read`, `write`, or `admin`.
+
+### Member
+
+Members are simply members of your organization. Owners can organize members into team and restrict their permissions via Access.
```

Comment:
- minus "simply."
- "into team" -> "into teams"
- "...and restrict their permissions. See the \[documentation on access controls\]\(./access.html\) for details."